packages/auth/src/accessToken/routes.ts

```diff
@@ -79,10 +81,30 @@ async function revokeToken(
 ): Promise<void> {
   //TODO: verify accessToken with httpsig method
 
-  const revocationError = await deps.accessTokenService.revoke(ctx.params['id'])
-  if (revocationError) {
-    return ctx.throw(404, revocationError.message)
+  await deps.accessTokenService.revoke(ctx.params['id'])
+  ctx.status = 204
+}
+
+async function rotateToken(
+  deps: ServiceDependencies,
+  ctx: AppContext
+): Promise<void> {
+  //TODO: verify accessToken with httpsig method
+  const result = await deps.accessTokenService.rotate(
+    ctx.params['managementId']
+  )
+  if (result.success) {
+    ctx.status = 200
+    ctx.body = {
+      access_token: {
+        access: [accessToBody(result.access)],
```

Comment:
I think you need to use a map function here, don't you? `result.access` is an array of accesses.